### PR TITLE
Feat/code of conduct

### LINKED ISSUE #40: feat: code of conduct
Create a CODE_OF_CONDUCT.md file in the root directory. You can get the generated version at [contributor covenant](https://www.contributor-covenant.org/#contributor-covenant-v21) (ps. you might want to read through the page) and click on the English Markdown version. Create a new branch conventionally (feat/code-of-conduct) and open a PR to pre-release. Happy coding 🎉

### PR DESCRIPTION
**Pull request by mettad3v**
Closes #40

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official e-mail address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at
+[https://discord.gg/duCVQt5VSU].
```
Comment (egdiala) [inline CODE_OF_CONDUCT.md:63]:
```suggestion
[https://discord.gg/duCVQt5VSU][NenLabs Discord Server].
```

**Review by egdiala — CHANGES_REQUESTED** [review]
@mettad3v I added a suggestion here

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (mettad3v) [reply CODE_OF_CONDUCT.md]: ok thanks

**Review by egdiala — APPROVED** [review]
LGTM 🚀

**Pull request merged**